### ISSUE TITLE
[alert] the http output chain is empty while sending to client upstream fastcgi

### ISSUE THREAD

**Issue by droplinxuser** [yaoweibin/ngx_http_substitutions_filter_module#47]
Those alerts/errors are produce with this module is enabled on some PHP pages.
Using Nginx 1.25.1 on Ubuntu 22.04.

**Comment by droplinxuser** [comment]
Complete log:
[alert] 131332#131332: *218555 the http output chain is empty while sending to client, client: 1.2.3.4, server: site.org, request: "GET /file.php HTTP/2.0", upstream: "fastcgi://127.0.0.1:9000", host: "site.org"

**Comment by yaoweibin** [comment]
Is there any problem when you see such alerts?

droplinxuser ***@***.***> 于2023年9月6日周三 08:55写道：

**Comment by droplinxuser** [comment]
Yes, when that happens, the page doesn't finish loading the whole page; it just sticks at loading and fails after several seconds.

With my thousand PHP-powered pages, it happens only on one page, though.
What I do for now is use subs_filter_bypass to disable running this module on that specific page.
